README.md

```diff
@@ -113,7 +113,7 @@ npm version [<new version> | major | minor | patch] -m "Release %s"
 
 ## License
 
-MIT
+SEE LICENSE IN LICENSE
```

Comment:
The readme is supposed to provide a summary of the project, so it seems more useful to show the license name than the NPM convention for pointing to license files. I'd sooner make this change on package.json than in here.